website/app/templates/design.hbs

```diff
@@ -73,7 +73,7 @@
           </ul>
         </p>
       </div>
-      {{!-- TODO: fix image sizing --}}
+      {{! TODO: fix image sizing }}
```

Comment:
This is kinda weird cause I think this is the default if you command - slash, but no big deal

Reply:
I had the same issue when adding a comment in HBS, and the linter complained. not sure which one of the two is the right one

Reply:
filed an issue, haven't tried it yet but may be a better way of using prettier in .hbs: https://github.com/hashicorp/flight/issues/371